### PR TITLE
Register member

### PR DESCRIPTION
**Pull request by DinolanNaidoo**

### CODE REVIEW

File: skietbaan_fe/src/components/Documents.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    constructor(props) {
+        super(props);
+        this.state = {
+            array: []
+        }
+        this.GetMembers = this.GetMembers.bind(this);
+    }
+
+    GetMembers() {
+        fetch("https://api.skietbaan.retrotest.co.za/api/Features/SearchMember", {
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/Documents.js]: Use BASE_URL constants please

File: skietbaan_fe/src/components/Documents.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        });
+        let name = this.state.array[user].username;
+        let email = this.state.array[user].email;
+        fetch("https://api.skietbaan.retrotest.co.za/api/Features/Search?Username=" + name, {
+            method: 'Get',
+            headers: { 'Accept': 'application/json', 'Content-Type': 'application/json' },
+        }).then(function (response) {
+            return response.json();
+        })
+            .then(function (data) {
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/Documents.js]: Change to `}).then(function (data) {`

File: skietbaan_fe/src/components/RegisterMember.js [hunk 3]
```diff
@@ -40,107 +43,136 @@ class App extends Component {
...
-              <Button onClick={this.UpdateMember} >Submit</Button>
-            </Col>
-          </Form>
+              </div>
+              <br />
+            </div>
+            <div className="expiry-date-member">
+              <Label><b>Membership Expiry Date</b></Label><br />
+              <input type="date" className="expdate" id="expdate" value={this.GetDate()} onChange={this.handleChange} />
+              <br />
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/RegisterMember.js]: Make use of styles instead of `<br />`

Comment (DinolanNaidoo) [reply skietbaan_fe/src/components/RegisterMember.js]: Have made the changes

File: skietbaan_fe/src/components/Documents.js (hunk 2 above)
Comment (DinolanNaidoo) [reply skietbaan_fe/src/components/Documents.js]: Done

File: skietbaan_fe/src/components/Documents.js (hunk 1 above)
Comment (DinolanNaidoo) [reply skietbaan_fe/src/components/Documents.js]: have done that

File: skietbaan_fe/src/components/Documents.css [hunk 4]
```diff
@@ -0,0 +1,26 @@
+.documents-container{
+    background-color: #171F32
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/Documents.css]: Missing `;`

Comment (FredJEhlers) [reply skietbaan_fe/src/components/Documents.css]: in a few places

File: skietbaan_fe/src/components/Documents.js [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+            .catch(function () { });
+    }
+
+    SearchMember(user) {
+        this.setState({
+            usernameValue: user
+        });
+        let name = this.state.array[user].username;
+        let email = this.state.array[user].email;
+        fetch("https://api.skietbaan.retrotest.co.za/api/Features/Search?Username=" + name, {
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/Documents.js]: Make sure to use BASE_URL every where

File: skietbaan_fe/src/components/RegisterMemberStyles.css [hunk 6]
```diff
@@ -6,36 +6,82 @@
...
+
+  .expiry-date-member{
+    padding-top: 80px;
+    position: center;
+    text-align: center
+  }
+
+  .create-member{
+    position: center;
+    text-align: center
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/RegisterMemberStyles.css]: more missed `;`

Comment (FredJEhlers) [reply skietbaan_fe/src/components/RegisterMemberStyles.css]: check all css files please

File: skietbaan_fe/src/components/ViewMembers.js [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+            <Table striped hover condensed className="table-member">
+                <thead>
+                    <tr>
+                        <td><b>Member</b></td>
+                        <td><b>Membership Expiry Date</b></td>
+                    </tr>
+                </thead>
+                <tbody>
+                    {this.state.array.filter((post) => {
+                        return (!this.state.filterText || post.username.toLowerCase().startsWith(this.state.filterText.toLocaleLowerCase()) || post.email.toLowerCase().startsWith(this.state.filterText.toLocaleLowerCase()))
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/ViewMembers.js]: Split over multiple lines

File: skietbaan_fe/src/components/Documents.css (hunk 4 above)
Comment (DinolanNaidoo) [reply skietbaan_fe/src/components/Documents.css]: done

File: skietbaan_fe/src/components/Documents.js (hunk 5 above)
Comment (DinolanNaidoo) [reply skietbaan_fe/src/components/Documents.js]: done

File: skietbaan_fe/src/components/ViewMembers.js (hunk 7 above)
Comment (DinolanNaidoo) [reply skietbaan_fe/src/components/ViewMembers.js]: done

**Pull request merged**